client/src/routes/profile/index.js

```diff
@@ -41,6 +42,8 @@ export default class Profile extends Component {
 					{' '}
 					Clicked {count} times.
 				</p>
```

Comment:
Let's go ahead and remove the click counter while we are here too it is just left over from the preact-cli example.